.catalog-onboard-pipeline.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: v1
+offerings:
+  - name: deploy-arch-ibm-access-management
+    kind: solution
+    catalog_id: 7df1e4ca-d54c-4fd0-82ce-3d13247308cd
+    offering_id: 48d2a13c-f353-47cb-a9c4-6fb560190ccc
+    variations:
+      - name: standard
+        mark_ready: false
+        install_type: fullstack
+        scc:
+          # Instance ID in account "GoldenEye FSCloud Testing (ae14871c179f49869e19657b37ec7c4e) <-> 2626044
```

Comment:
This repo will us the GEdev account, so remove this comment and set instance id to `1c7d5f78-9262-44c3-b779-b28fe4d88c37`